sql/core/src/main/scala/org/apache/spark/sql/execution/ExistingRDD.scala

```diff
@@ -114,6 +114,7 @@ private[sql] object PhysicalRDD {
   // Metadata keys
   val INPUT_PATHS = "InputPaths"
   val PUSHED_FILTERS = "PushedFilters"
+  val HANDLED_FILTERS = "HandledFilters"
```

Comment:
`HandledFilters` at here means filters that will be applied to every row inside the data source, right? Is there a better name?